test/units/modules/network/nxos/test_nxos_bgp_neighbor_af.py

```diff
@@ -97,3 +97,11 @@ def test_nxos_bgp_neighbor_af_max_prefix(self):
             changed=True, sort=False,
             commands=['router bgp 65535', 'neighbor 3.3.3.5', 'address-family ipv4 unicast', 'maximum-prefix 20 20']
         )
+
+    def test_nxos_bgp_neighbor_af_disable_peer_as_check(self):
+        set_module_args(dict(asn=65535, neighbor='3.3.3.5', afi='ipv4',
+                             safi='unicast', disable_peer_as_check=True))
+        self.execute_module(
+            changed=True, sort=False,
```

Comment:
You shouldn't need `sort=False` here... there's only one command (disable-peer-as-check)